### DISCUSSION TITLE
writing/2024/06/05/predictions-for-the-future-of-rag/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#85] [General]
# writing/2024/06/05/predictions-for-the-future-of-rag/

Explore the future of RAG in report generation, enhancing decision-making and resource allocation for businesses.

https://jxnl.co/writing/2024/06/05/predictions-for-the-future-of-rag/

**Comment by ShreeshaBhat1004** [comment]
With increased context window size, RAG technique will loose its significance

**Reply by jxnl** [reply]
but the SOP will stay

**Comment by parthatom** [comment]
We probably need a new term for the type of RAG required for reports. Diff from Q&A RAG is Q&A because of it's dynamic nature has to rely heavily on the retrieval part. With reports, you (usually) know which part of your sources you can find certain information from (you're report clarity will greatly benefit from you knowing the source). I like to think of it as static RAG but it's just enhanced prompting.

**Comment by paoloap-py** [comment]
Which book the sales letter template came from? I'd be curious to learn more about that.

**Comment by clintoncunningham** [comment]
Basically RAG + Agents
